Computer History Museum Archive.js

```diff
@@ -0,0 +1,988 @@
+				"archive": "Computer History Museum",
+				"archiveLocation": "X2595.2004",
+				"extra": "Catalog Number: 102706809\nType: Audio\nOther number: camvchm_000022\nOther number: Tape #8\nFormat: Standard audio cassette\nSeries Title: The First West Coast Computer Faire\nCredit: Gift of Jim Warren",
+				"libraryCatalog": "Computer History Museum Archive",
+				"place": "San Francisco, CA",
+				"publisher": "Butterfly Media Dimensions",
+				"attachments": [
+					{
+						"mimeType": "audio/mpeg",
+						"title": "Tutorials on software systems design - Side B"
+					},
+					{
+						"mimeType": "audio/mpeg",
+						"title": "Tutorials on software systems design - Side A"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102661095",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "People's Computer Company; People's Computers; Recreational Computing",
+				"creators": [],
+				"date": "1972-1981",
+				"abstractNote": "Founded in Oct 1972 as a large-format bimonthly newsprint publication called \"People's Computer Company.\" Name changed to \"People's Computers\" beginning with the May-June 1977 issue and the format changed to a more conventional magazine style, albeit with uncoated paper. Name changed again to \"Recreational Computing\" with Jan-Feb 1979 issue with slicker covers and paper. \n\nNote: The print collection has been augmented with scans provided by Bob Zeidman under lot number X6691.2013. Some issues in the collection exist in digital form only.",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X2595.2004",
+				"extra": "Catalog Number: 102661095\nType: Document\nCategory: Periodical\nCredit: Gift of Jim Warren",
```

Comment:
`Type` shouldn't be in any of these. If these items aren't books, they shouldn't have the Book item type. They should be saved as appropriate item types. This may require a mapping between the CHM types and Zotero item types.

Reply:
I see! Can’t believe I forgot about this. Let me restructure the program to tell the Zotero item type first.

Reply:
Fixed.